scripts/clash.tool

```diff
@@ -135,58 +136,61 @@ update_file() {
 
         if [ -f ${file} ]; then
             mv -f ${file} ${file_bak}
-            echo "warning msg= backup file ${file_bak}" >> ${CFM_logs_file}
+            echo $date_log"warn: backup file ${file_bak}" >> ${CFM_logs_file}
         fi
         echo "curl -L -A 'clash' ${update_url} -o ${file} "
```

Comment:
`echo "curl -k --insecure -L -A 'clash' ${update_url} -o ${file} "`
`curl -k --insecre -L -A 'clash' ${update_url} -o ${file} 2>&1`

_Allow insecure connection for online config_